client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -763,6 +771,21 @@ class EmailProvidersComparison extends Component {
 		const shouldShowEmailForwardWarning =
 			! hideEmailForwardingCard && ! isSubmittingEmailForward && ! this.state.emailForwardAdded;
 
+		const googleCard = this.renderGoogleCard();
+		const titanCard = this.renderTitanCard();
+
+		const paidCards = shouldPromoteGoogleWorkspace ? (
```

Comment:
Nit: It may be better to either inline this or export the logic into a method. It would feel easier to read, and the components are rendered at the right time.

Reply:
I decided not to switch the order, because I am concerned too many people won't see the free option when the Google card is placed first and expanded.